CHANGELOG/CHANGELOG-1.7.md

```diff
@@ -0,0 +1,109 @@
+  * Refer to [this document](https://github.com/antrea-io/antrea/blob/release-1.7/docs/traffic-control.md) for more information about this feature.
+  * Refer to [this cookbook](https://github.com/antrea-io/antrea/blob/release-1.7/docs/cookbooks/ids/README.md) for more information about using this feature to provide network-based intrusion detection service to your Pods.
+- Add support for the IPsec Certificate-based Authentication. ([#3778](https://github.com/antrea-io/antrea/pull/3778), [@xliuxu])
+  * Add an Antrea Agent configuration option `ipsec.authenticationMode` to specify authentication mode. Supported options are "psk" (default) and "cert".
+  * Add an Antrea Controller configuration option `ipsecCSRSigner.autoApprove` to specify the auto-approve policy of Antrea CSR signer for IPsec certificates management. By default, Antrea will auto-approve the CertificateSingingRequest (CSR) if it is verified.
+  * Add an Antrea Controller configuration option `ipsecCSRSigner.selfSignedCA` to specify whether to use auto-generated self-signed CA certificate. By default, Antrea will auto-generate a self-signed CA certificate.
+- Add the following capabilities to Antrea-native policies:
+  * Add support for matching ICMP traffic. ([#3472](https://github.com/antrea-io/antrea/pull/3472), [@GraysonWu])
+  * Add support for matching multicast and IGMP traffic. ([#3660](https://github.com/antrea-io/antrea/pull/3660), [@liu4480])
+  * Add support for rule-level statistics for multicast and IGMP traffic. ([#3449](https://github.com/antrea-io/antrea/pull/3449), [@ceclinux])
+- Add the following capabilities to the Multicast feature:
+  * Add `antctl get podmulticaststats` command to query Pod-level multicast traffic statistics in Agent mode. ([#3449](https://github.com/antrea-io/antrea/pull/3449), [@ceclinux])
+  * Add "MulticastGroup" API to query Pods that have joined multicast groups; `kubectl get multicastgroups` can generate requests and output responses of the API. ([#3354](https://github.com/antrea-io/antrea/pull/3354) [#3449](https://github.com/antrea-io/antrea/pull/3449), [@ceclinux])
+  * Add an Antrea Agent configuration option `multicast.igmpQueryInterval` to specify the interval at which the antrea-agent sends IGMP queries to Pods. ([#3819](https://github.com/antrea-io/antrea/pull/3819), [@liu4480])
+- Add the following capabilities to the Multi-cluster feature:
+  * Add Gateway and ClusterInfoImport CRDs and controllers to allow basic cluster information (Service CIDR, Gateway IPs etc.) exchange between member clusters. ([#3689](https://github.com/antrea-io/antrea/pull/3689) [#3463](https://github.com/antrea-io/antrea/pull/3463),  [@luolanzone])
```

Comment:
I would merge the two points to one: Add the Multi-cluster Gateway functionality which supports routing multi-cluster Service traffic across clusters though tunnels between the Gateway Nodes. It enables multi-cluster Service access across clusters, without requiring direct reachability of Pod IPs between clusters.

It is very hard for readers to understand what the feature is for if we just list the individual changes.

Reply:
done, thanks